source/common/config/utility.cc

```diff
@@ -113,25 +113,25 @@ void Utility::checkApiConfigSourceNames(
 
   if (is_grpc) {
     if (!api_config_source.cluster_names().empty()) {
-      throw EnvoyException(fmt::format("envoy::api::v2::core::ConfigSource::(DELTA_)GRPC "
+      throw EnvoyException(fmt::format("envoy::api::v3::core::ConfigSource::(DELTA_)GRPC "
```

Comment:
Is it possible to use `GetTypeName()` (https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.message#Message) to get the protobuf type name without harcoding the version?

Reply:
I was thinking about this but did not know the exact method. Let me try this